src/app/components/organism/timeline-element/timeline-element.component.css

```
@@ -1,324 +1,333 @@
+  /* glass-card surface — light blur keeps it crisp "crystal", not foggy */
+  background: hsla(263, 50%, 14%, 0.35);
+  backdrop-filter: blur(1px) saturate(1.2);
+  -webkit-backdrop-filter: blur(1px) saturate(1.2);
+  border: 1px solid hsla(270, 95%, 83%, 0.06);
+  box-shadow: var(--shadow-glass);
+  transition:
+    transform 0.4s cubic-bezier(0.16, 1, 0.3, 1),
+    border-color 0.45s cubic-bezier(0.23, 1, 0.32, 1),
+    box-shadow 0.45s cubic-bezier(0.23, 1, 0.32, 1);
+
+  transition: transform var(--dur-base) var(--ease-out-soft), box-shadow var(--dur-base) var(--ease-out-soft);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)